### ISSUE TITLE
WebVTT: yay or nay?

### ISSUE THREAD

**Issue by sandersaares** [Dash-Industry-Forum/DASH-IF-IOP#289]
IOP v4.3 does not clearly say whether WebVTT is something we consider interoperable or not. It does mention WebVTT under "timing for side-loaded files" but appears otherwise not mentioned.

As I have seen it used in real world, I thought I would raise the topic. Do we want to encourage or discourage its use? Should we list it as an interoperable technology?

As far as I know, WebVTT is quite well supported by players, so I am in favor of including it in our media coding technologies chapter alongside IMSC1.

**Comment by RufaelDev** [comment]
Different companies/regions/cultures use different timed text schemes, i don't think it is fair to let everybody convert their subtitles to IMSC1, supporting webvtt sounds fair to me as it is well supported in practice and also well documented (MPEG-4 part 30, CMAF) etc. However we may want to add support for Webvtt in ISOBMFF (MPEG-4 part 30) for consistency with CMAF. The side loaded WebVTT and IMSC1 seems already part of the IOP as is mentioned already. So what really should be added is the WebVTT in CMAF/ISOBMFF.

**Comment by haudiobe** [comment]
I agree that we should make it clearer. If WebVTT is used, we should ensure interop.
